react-backend/client/src/components/visitors/form_privacy2.js

```diff
@@ -1,25 +1,21 @@
-import React, { Component } from 'react';
+import React from 'react';
 
-export class FormPrivacy2 extends Component {
-  render() {
-    return (
-      <aside className="Privacy col-3">
-        <img className="Privacy__image" src="/images/question.png" alt="question" />
-        <h3 className="Privacy__title">What happens to the data I disclose here?</h3>
-        <p className="Privacy__text">
-          We will use that data to analyse trends and preferences of our customers, and to improve
-          our services in line with those.
-          <br />
-          <br />
-          We will not use your data to send you marketing material. We only use your data in full
-          compliance with the Data Protection Act and the General Data Protection Regulation.
-          <br />
-          <br />
-          Only the senior leadership of this organisation can see data on your visits. The
-          independent trust Power to Change will access an aggregated version of your data, to
-          conduct analysis that will help businesses like ourselves thrive.
-        </p>
-      </aside>
-    );
-  }
-}
+export default () => (
+  <aside className="Privacy col-3">
+    <img className="Privacy__image" src="/images/question.png" alt="question" />
```

Comment:
Is it worth having a more descriptive alt text here?

Reply:
Like what? "question mark"?

Reply:
Yeah pretty much, "question" is not very helpful for a screenreader, but this is super minor and can be done whenever.